discourse.tf

```diff
@@ -8,6 +8,7 @@ module "discourse-production" {
     elasticache_subnet_group            = "${aws_elasticache_subnet_group.elasticache-production-subnet-group.name}"
     fqdn                                = "discourse.mozilla-community.org"
     ssl_certificate                     = "${lookup(var.ssl_certificates, "community-sites-elb-${var.aws_region}")}"
+    aws_account_id                      = "${var.aws_account_id}"
```

Comment:
Not sure what you mean by new build, but it won't recreate any resources, only add new policies/attachments